login_password_page.php

```diff
@@ -170,7 +170,7 @@
 }
 
 $t_upgrade_required = false;
-if( config_get_global( 'admin_checks' ) == ON && file_exists( dirname( __FILE__ ) .'/admin' ) ) {
+if( config_get_global( 'admin_checks' ) == ON && file_exists( dirname( __FILE__ ) .'/admin/.' ) ) {
```

Comment:
would using dir exist make sense here?

Reply:
What do you mean with `dir exist`? I didn't find such a function or a similar one, like  `dir_exists`.

`file_exists` should do the job according http://php.net/manual/en/function.file-exists.php
> Checks whether a file or **directory** exists.

Reply:
I was wondering if dir_exists removes the need for `/.` at the end of the folder name.

Reply:
> I was wondering if dir_exists ...

As I said, I didn't find such a function.
Can you provide a link to the documentation?

Reply:
The use of trailing `/.` is correct, and to my knowledge the only way to do this check properly.

> > I was wondering if dir_exists ...
> As I said, I didn't find such a function.

Does not exist in PHP. It's either _file_exists()_ or _is_dir()_

Reply:
I meant is_dir(), but if others confirmed that this is fine, I'm ok with it.